imports/ui/AddTask.js

```diff
@@ -35,61 +36,82 @@ export class AddTask extends React.Component {
     this.setState({ dueDate });
   };
 
+  // eslint-disable-next-line no-undef
+  handleSubmitTaks = () => !!this.state.description;
+
   // eslint-disable-next-line no-undef
   addTaskAndGo = () => {
-    const { router: { history } } = this.context;
-    const { addTask } = this.props;
-    addTask({
-      variables: {
-        task: {
-          ...this.state,
+    if (this.handleSubmitTaks()) {
+      const { addTask } = this.props;
+      addTask({
+        variables: {
+          task: {
+            _id: this.state._id,
+            description: this.state.description,
+            details: this.state.details,
+            dueDate: this.state.dueDate,
+            done: this.state.done,
+          },
         },
-      },
-    }).then(({ data: { addTask: { _id } } }) => {
-      if (_id) {
-        history.push('/');
-      }
+      })
+        .then(() => {})
+        .catch(error => {
+          // eslint-disable-next-line no-console
+          console.log(error);
+        });
+      document.getElementById('bottomNavigationActionTasks').click();
```

Comment:
We need to use history.push to change URL, and we should never use document.something with React. That's very rare.